skypy/galaxy/luminosity.py

```diff
@@ -4,15 +4,158 @@
 
 import numpy as np
 
-import skypy.utils.astronomy as astro
 from skypy.utils.random import schechter
+from skypy.utils import uses_default_cosmology
 
 
 __all__ = [
+    'absolute_to_apparent_magnitude',
+    'apparent_to_absolute_magnitude',
+    'distance_modulus',
+    'luminosity_in_band',
+    'luminosity_from_absolute_magnitude',
+    'absolute_magnitude_from_luminosity',
     'herbel_luminosities',
 ]
 
 
+def absolute_to_apparent_magnitude(M, DM):
```

Comment:
```suggestion
def absolute_to_apparent_magnitude(absolute_magnitude, distance_modulus=None, redshift=None, cosmology=None):
    if not distance_modulus:
        distance_modulus = skypy.distance_modulus(redshift, cosmology)
```
I think we need to support this conversion given a redshift so that pipeline users don't have to calculate the distance modulus as an intermediate column. Could use this suggestion or implement an additional function

Reply:
Also use full names for variables rather than abbreviations or symbolic representations where practical

Reply:
My reasoning was that one would always want to keep the distance modulus around after computation, because it takes a while to compute, and I don't like either-or function signatures. If a user wants to not keep the distance modulus around and compute it in one go, they can chain the function call.

Reply:
And `absolute_magnitude` and `distance_modulus` are way too long words to force anyone to type them out...

Reply:
But later in the use case, the user don't really have to type them out.